pay-api/src/pay_api/services/refund.py

```diff
@@ -133,7 +133,9 @@ def create_refund(cls, invoice_id: int, request: Dict[str, str], **kwargs) -> No
         # Do validation by looking up the invoice
         invoice: InvoiceModel = InvoiceModel.find_by_id(invoice_id)
         # Allow refund only for direct pay payments, and only if the status of invoice is PAID/UPDATE_REVENUE_ACCOUNT
-        paid_statuses = (InvoiceStatus.PAID.value, InvoiceStatus.UPDATE_REVENUE_ACCOUNT.value)
+        paid_statuses = (
```

Comment:
Adding APPROVED as well for refunds. APPROVED will be refunded, but no credit memo created.